examples/features/src/ray_shadows/shader.wgsl

```diff
@@ -33,7 +33,10 @@ var<uniform> uniforms: Uniforms;
 @group(0) @binding(1)
 var acc_struct: acceleration_structure;
 
-var<push_constant> light: vec3<f32>;
+struct PushConstants {
+    light: vec3<f32>,
+}
+var<push_constant> pc: PushConstants;
```

Comment:
question: What's the issue number for this? You mention in the commit that we're aware of this issue, but having the link would be good.

Reply:
https://github.com/gfx-rs/wgpu/issues/5683

Reply:
I'm not sure that that bug is relevant. It talks about `mat4x4` in the title and repro, not `vec3f`. 🤔

Either way, if you're confident that that's the bug, could we explain why this is necessary and point to it with a comment, pretty please? 🥺

Reply:
It is that bug, that bug is should be any non-structure types, will rename.